supertokens_python/recipe/session/framework/fastapi/__init__.py

```diff
@@ -22,11 +22,12 @@
 from supertokens_python.exceptions import SuperTokensError
 from supertokens_python.types import MaybeAwaitable
 from fastapi.responses import JSONResponse
-from fastapi import Request
 
 from ...interfaces import SessionContainer, SessionClaimValidator
 from supertokens_python.utils import set_request_in_user_context_if_not_defined
 
+from fastapi import Request
```

Comment:
why is this imported twice in this file?

Reply:
Fixed.